smart-campus-service-impl/src/main/java/hu/unideb/smartcampus/service/api/impl/MessageProcessingServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package hu.unideb.smartcampus.service.api.impl;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import hu.unideb.smartcampus.service.api.MessageProcessingClass;
+import hu.unideb.smartcampus.service.api.MessageProcessingService;
+import hu.unideb.smartcampus.service.api.context.MessageProcessContext;
+import hu.unideb.smartcampus.service.api.domain.response.wrapper.BaseWrapper;
+import hu.unideb.smartcampus.shared.exception.ProcessMessageException;
+import hu.unideb.smartcampus.shared.requestmessages.BaseRequestType;
+
+/**
+ * Implementation of the message processing service.
+ *
+ */
+@Service
+public class MessageProcessingServiceImpl implements MessageProcessingService {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MessageProcessingServiceImpl.class);
+
+  private ObjectMapper objectMapper;
+
+  private MessageProcessContext context;
+
+  private ApplicationContext appContext;
+
+  /**
+   * Constructs a MessageProcessingServiceImpl instance with injected dependencies.
+   *
+   */
```

Comment:
Yeah, it missing but I was using this for a lot of case, and worked well :D I am gonna take it back.